experiments/Atmos_LES/dycoms.jl

```diff
@@ -375,7 +380,17 @@ function main()
     timeend = FT(100)
 
     driver_config = config_dycoms(FT, N, resolution, xmax, ymax, zmax)
-    solver_config = CLIMA.setup_solver(t0, timeend, driver_config, init_on_cpu=true)
+
+    # Courant number
+    if driver_config.solver_type.solver_method == LSRK144NiegemannDiehlBusch
+        CFL_max = FT(2.4)
+    elseif driver_config.solver_type.solver_method == LSRK54CarpenterKennedy
+        CFL_max = FT(0.6)
+    else
+        CFL_max = FT(0.5)
+    end
```

Comment:
There is a lot of unnecessary code repetition introduced here. Let's try something a bit different. I think this check here can be part of the constructor for whatever `AbstractSolverType` comes in. For example, consider something like this in `src/Driver/Configurations.jl`:
```julia
struct ExplicitSolverType <: AbstractSolverType
    solver_method::Function
    courant_number::FT

    function ExplicitSolverType(;solver_method  = LSRK54CarpenterKennedy,
                                 courant_number = nothing)
        # User has specified a specific Courant number to use
        if courant_number !== nothing
            return new(solver_method, courant_number)
        # Otherwise, estimate a stable Courant number
        else
            courant_number = solver_method.estimated_CFL
            return new(solver_method, courant_number)
    end
end
```
You'll need to attach an `estimated_CFL` property to the desired ode solver. Similarly, you can update `IMEXSolverType` in the same way. Not sure if this is the best way, but hopefully the idea is clear.

Now we no longer need to duplicate this piece of code to configure the timestepper. And it allows users to bypass this check entirely and just specify their own Courant number. Now you can change lines 383--391 here to simply
```julia
CFL_max = driver_config.solver_type.courant_number
```
Just my thoughts.

Reply:
If the rising bubble is fixed (e.g, so the advection velocity is not so high) default will likely work better. Right now DYCOMS with LSRK14 can run with CFL of 2.4 whereas rising bubble and surface bubble are 1.7. I believe that this is why @skandalaCLIMA opted to have a different CFL for each.

Reply:
@thomasgibson and @fxgiraldo: Does surface bubble have the same high advection velocity problem that the current version of rising bubble does?

Reply:
Not sure about the surface bubble.

Reply:
> If the rising bubble is fixed (e.g, so the advection velocity is not so high) default will likely work better. Right now DYCOMS with LSRK14 can run with CFL of 2.4 whereas rising bubble and surface bubble are 1.7. I believe that this is why @skandalaCLIMA opted to have a different CFL for each.

Oh, I agree! I was just proposing a possible alternative. And there may be instances where we know that we can push the Courant number out further than the default (acoustic) Courant (multirate, for example).

Reply:
For there to be a `solver_method.estimated_CFL` is how I imagined this -- agree with this suggestion.

Reply:
> If the rising bubble is fixed (e.g, so the advection velocity is not so high) default will likely work better. Right now DYCOMS with LSRK14 can run with CFL of 2.4 whereas rising bubble and surface bubble are 1.7. I believe that this is why @skandalaCLIMA opted to have a different CFL for each.

@jkozdon with what code version or PR were you able to run dycoms at Courant = 2.4? I can only keep it stable for up to 1.8 with LSRK14. 
Let me know and I'll try that. Thanks

Reply:
> @jkozdon with what code version or PR were you able to run dycoms at Courant = 2.4?

I have no idea. This is just what @skandalaCLIMA put into the driver.

Reply:
Hello @smarras79, I was able to run the simulation in experiments/Atmos_LES/dycoms.jl to completion (timeend = 100; dt = 9.78474e-02) with a CFL of 2.4 for LSRK14. Would you recommend a different timeend for this simulation?